src/common/utility.js

```diff
@@ -360,7 +361,53 @@ export const unslugify = (slug) => {
         })
     }
 }
+export const removeSpecialCharacterUrl = (url) =>
+    url &&
+    slugify(url)
+        .replace(/\?+/g, '') // Replace question mark with empty value
+        .replace(/[/]/g, '-') //Replace '/' with single -
+
+export const queryParams = {
+    get: (key) => {
+        const params = new URLSearchParams(location.search)
+        let param_values = {}
+        //To get the params from the url
+
+        if (typeof key === 'string') {
+            console.log('params', params.get(key))
```

Comment:
Please remove the console. if it is not necessary

Reply:
okay removed.thanks @ashraf-deriv